spec/factories.rb

```diff
@@ -16,4 +16,9 @@
       encrypted_password ''
     end
   end
+
+  factory :person do
+    email
+    password 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.